pkg/database/database.go

```diff
@@ -39,12 +39,17 @@ func (db *DbInstance) DSN() string {
 	return db.dsn
 }
 
+// DB is a global db instance.
 var DB *DbInstance = &DbInstance{}
 
+// ConnectDB will establish and return a DbInstance while also caching it in the global
+// var database.DB.
 func ConnectDB(
```

Comment:
maybe we introduce a second method here that does not override the global DB instance, and call it from `ConnectDB`?
when the Aim refactoring is completed, the global DB instance will be gone and instead passed down from the `server` command to the various components

Reply:
okay, will do